Database/Table Scripts/CreatePetTableScript.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE dbo.Pet(
+	Id BIGINT PRIMARY KEY IDENTITY(1,1) NOT NULL,
+	OwnerId BIGINT NOT NULL FOREIGN KEY REFERENCES dbo.Client(Id),
+	Name VARCHAR(50) NOT NULL,
+	Breed VARCHAR(50) NOT NULL,
```

Comment:
Let's make a separate table for Breed actually.

Breed will have like Id, BreedName, and Type (dog/cat).

Then link this with foreign key.

Then if you create a new table, make sure create REPO for that table (to be use for drop down).

Add method like GetBreedByType - returns all breed related to dog or cat when selected.

Add basic add/update by id/delete by id functionality (for admin use only)